src/modelfree/shaping_wrappers.py

```diff
@@ -186,8 +153,30 @@ def load(self):
         raise NotImplementedError()
 
 
+def load_wrapper_params(params_path, env_name, rew_shaping=False, noisy_victim=False):
+    config_dir = 'rew_configs' if rew_shaping else 'noise_configs'
+    path_stem = os.path.join('experiments', config_dir)
+    default_configs = {
+        'multicomp/SumoHumans-v0': 'default_hsumo.json',
+        'multicomp/SumoHumansAutoContact-v0': 'default_hsumo.json'
+    }
+    fname = os.path.join(path_stem, default_configs[env_name])
+    with open(fname) as default_config_file:
+        final_params = json.load(default_config_file)
+    if params_path != 'default':
+        with open(params_path) as config_file:
+            config = json.load(config_file)
+        if rew_shaping:
+            final_params['sparse'].update(config.get('sparse', {}))
```

Comment:
Is the only reason you need to special case for `rew_shaping` because it has nested dicts? You could write a recursive implementation that merges possibly-nested `dict`'s. Probably not worth it right now, but would consider it if you have more configs in the future.